commercetools-models/src/main/java/io/sphere/sdk/cartdiscounts/StackingMode.java

```diff
@@ -0,0 +1,23 @@
+package io.sphere.sdk.cartdiscounts;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.sphere.sdk.models.SphereEnumeration;
+
+/**
+ * Describes how this discount interacts with other discounts.
+ */
+public enum StackingMode implements SphereEnumeration {
+    /**
+     * Default. Continue applying other matching discounts after applying this one.
+     */
+    STACKING,
+    /**
+     * Don't apply any more matching discounts after this one.
+     */
+    STOP_AFTER_THIS_DISCOUNT;
```

Comment:
First time i discover how `SphereEnumerations` are mapped to json.
😄 interesting 👍